api/utils/requestProcessor.js

```diff
@@ -273,12 +273,12 @@ const processRequest = (params) => {
                 }
                 switch (paths[3]) {
                 case 'save':
-                    validateUserForWriteAPI(params, () => {
+                    validateCreate(params, 'core', () => {
```

Comment:
I think this works both as create and update, right? so we need to decide what to do in those situations
but my guess we would go with update right

Reply:
yes, nice catch. but if we go with "update" rights control in here, we can't allow users who have only "create" rights, so I think we need a custom solution here. maybe we can add some dependency rule for "core" permission that force user to give both "update" and "create" permission on the frontend. or custom permission check method like validateCreateOrUpdate (). but the first solution sounds better to me.

I'm leaving this part as it is, and I took notes for the new-UI permission box about this.